### PR TITLE
Add test for Breaking change RemovedSupportForJSPTemplates

### PR DESCRIPTION
**Pull request by Seiphon**

### CODE REVIEW

**Comment by jtydhr88** [comment]
SF Successfully on liferay-ide folder

**Comment by jtydhr88** [comment]
SF failed on liferay-ide, please run it locally

**Comment by Seiphon** [comment]
ci:retest

**Comment by Seiphon** [comment]
ci:test

**Comment by jtydhr88** [comment]
SF Successfully on liferay-ide folder

File: tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/SSEXMLFile.java [hunk 1]
```diff
@@ -178,14 +178,26 @@ public FileSearchResult findDocumentTypeDeclaration(String name, Pattern idPatte
 			for (int i = 0; i < elements.getLength(); i++) {
 				IDOMElement domElement = (IDOMElement)elements.item(i);
 
-				String classValue = domElement.getAttribute("class");
+				String attributeValue = null;
+				String attributeName = null;
 
-				Matcher matcher = pattern.matcher(classValue);
+				if (tagName.equals("div")) {
```
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/SSEXMLFile.java]: I don't think this class should have any particular tags

**Pull request closed without merging**